test/java/org/opendatakit/briefcase/reused/UncheckedFilesIsInstanceDirTest.java

```diff
@@ -55,13 +55,6 @@ public void a_file_is_not_an_instance_directory() {
     assertThat(isInstanceDir(someFile), is(false));
   }
 
-  @Test
-  public void an_arbitrary_named_directory_is_not_an_instance_directory() {
-    assertThat(isInstanceDir(createDirectories(tempDir.resolve("cocotero"))), is(false));
```

Comment:
Odd choice of variable name, but I guess this is a Spanish thing?

Reply:
Are you referring to "cocotero"? I can change that to something more neutral. I use cocotero when I don't want to think about a name and it doesn't really matter but I try to change that after. This one slipped through.

Reply:
OK, changed it. 

By the way, cocotero means coconut tree.